extensions/amp-render/1.0/base-element.js

```diff
@@ -16,6 +16,7 @@
 
 import {PreactBaseElement} from '../../../src/preact/base-element';
 import {Render} from './component';
+import {getJsonFn} from './amp-render';
```

Comment:
You can't import the amp-element into base-element. If the element needs AMP–mode specific behavior, that must be done inside the amp-element.js only.

Reply:
Does this mean we cannot do `'getJson': {attrs: ['src'], parseAttrs: getJsonFn}` and use `mutationObserverCallback` instead?

Reply:
You can reassign `props` in the amp-element file:

```js
// amp-render.js
AmpRender['props'] = {
  ...BaseElement['props'],
  'getJson': {attrs: ['src'], parseAttrs: getJsonFn},
}

Reply:
Done. Thanks for quick response 🙂